src/site/_includes/layout.njk

```diff
@@ -44,14 +44,12 @@
     <script defer src="//www.gstatic.com/firebasejs/6.6.1/firebase-performance.js"></script>
     {# This is used by src/lib/firestore-loader.js to dynamically load Firestore #}
     <meta id="firebase-firestore" value="//www.gstatic.com/firebasejs/6.6.1/firebase-firestore.js" />
-    <script>
-      window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
-      ga('create', '{{ site.analytics.ids.prod }}');
-      ga('set', 'transport', 'beacon');
-      ga('set', '{{ site.analytics.dimensions.TRACKING_VERSION }}', '{{ site.analytics.version }}');
-      ga('send', 'pageview');
-    </script>
     <script async src="//www.google-analytics.com/analytics.js"></script>
+    {#
+      The basic script is "defer" as it should run before the normal bootstrap module code (which
+      is defer by default).
+    #}
+    <script defer src="/basic.js"></script>
```

Comment:
wdyt about renaming the file to something more descriptive like `analytics`? although i think we already have an analytics.js elsewhere, but `basic` seems like folks wouldn't be able to tell at a glance what it does.